_posts/2020-04-26-fireside-chats.md

```diff
@@ -0,0 +1,18 @@
+---
+author: Mandy
+title: Fireside chats - winning hackathons and meeting an ER doctor
+---
+
+We have been having some awesome talks done by these experts recently:
+* Q and A with doctor/developer/designer Dr. Vincent Dumouchel
+* "How to Win a Hackathon" by Ben Zittlau
+
+## Q and A with doctor/developer/designer
+Dr. Vincent Dumouchel shared a lot with us about his career path through schooling, bootcamps and how he now has a part time gig as an ER doctor and as director of a tech agency. We talked about the difference about being on an ER team (think commanding and getting things done fast) and a tech team (think good communication and talking things through). We talked about one major need that most Canadian hospitals have is the need to monitor patients in waiting rooms because people can come in with a headache but then soon the symptoms of COVID-19 emerge and go from okay to ICU quite fast. We also talked about what a good leader is, whether that is at a company or on a hackathon team. Vincent said **"a leader is someone who makes space for everyone to talk. You need a leader on a hackathon team to organize everything and make sure everyone’s voice is heard"**. After the session was over we all commented on how warm it was to be able to talk to people about what we are going through. Vincent said it was important to **"take time to speak about your feelings"**
+
+[View recording here](https://drive.google.com/file/d/1d9DqFb6jf-LSQS3wMJIVzVOHmDICsI9d/view?usp=sharing)
+
+## "How to Win a Hackathon" by Ben Zittlau
+Ben gave us awesome tips on how to make the best of a Hackathon. See his slides (here)[https://tinyurl.com/how-to-win-a-hackathon]. One piece of encouragement Ben gave us is to **"Use this opportunity to practice.  Don’t wait until you believe you have the best project you’ve ever done, to do your first submission."** Hackathon's can be for fun and even if your project isn't "finished" take some time to try to submit for fun anyways! You never know, judges could love it.
```

Comment:
brackets are backwards [] first then () for the url-link was dead on deploy too.

Reply:
wow, look at you, an expert coder now

I'll fix lol

Reply:
Na, just a flair for the obvious!  I literally know two things about Markdown and you messed one up!  You should see me SQL-terrifying!